docker-compose-2.8.yml

```diff
@@ -81,7 +83,7 @@ services:
     build:
       context: ./src/2.8/ckan-mock-harvest-sources/static/
     ports:
-      - "11088:11088"
+      - "11089:11088"
```

Comment:
This won't work properly - it will mean the mock harvest source will be publishing links to data files that are served on the wrong port.

Reply:
It needs to correspond to whichever port is specified in https://github.com/alphagov/ckan-mock-harvest-sources/blob/07273f596227f4969eb15e0a0e2229a5cc909e9a/static/vars.conf#L3

Reply:
hmmm, we may need to find a work around as it's not possible to run 2 stacks side by side with the same port

Reply:
Perhaps running two stacks side by side is why your machine keeps running out of memory.

Reply:
But it should be fixable by changing the line

```
echo $'\nmap $host $mock_absolute_root_url { default "http://static-mock-harvest-source:11088/"; }' >> ckan-mock-harvest-sources/static/vars.conf
```

in `bootstrap.sh` to vary the port by version.

Reply:
I've put it back to the default setting, will figure out a way of running 2 stacks with static mock harvest source running